opentelemetry-tracing/opentelemetry-go/gorilla-mux/client/main.go

```diff
@@ -0,0 +1,77 @@
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"time"
+
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+
+	"github.com/signalfx/splunk-otel-go/distro"
+)
+
+func main() {
+	// handle CTRL+C gracefully
+	ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt)
+	defer stop()
+
+	// initialize Splunk OTel distro
+	sdk, err := distro.Run()
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err := sdk.Shutdown(context.Background()); err != nil {
```

Comment:
This uses a different context then the one linked to CTRL+C. This opens the possibility for the program to hang after the user sends the terminate signal right?

Reply:
It should not hang b/c of https://github.com/pellared/tracing-examples/blob/f8eeb776ccef55873f3f709e71f29d3ccb7126ed/opentelemetry-tracing/opentelemetry-go/gorilla-mux/server/main.go#L79-L80

Reply:
That link is for the server not the client(?). Am I missing something?

Reply:
No, you are correct. It is me who is missing something 😄

Reply:
Addressed https://github.com/signalfx/tracing-examples/pull/106/commits/bc78186192dea3ae4353fc7569584390ea25dd17